client/extensions/woocommerce/app/reviews/review-actions-bar.js

```diff
@@ -69,10 +83,40 @@ const ReviewActionsBar = ( { review, currentStatus, translate } ) => {
 };
 
 ReviewActionsBar.propTypes = {
+	siteId: PropTypes.number,
 	review: PropTypes.shape( {
 		status: PropTypes.string,
 	} ).isRequired,
 	currentStatus: PropTypes.string.isRequired,
+	toggleExpanded: PropTypes.func,
 };
 
-export default localize( ReviewActionsBar );
+const mapDispatchToProps = ( dispatch, ownProps ) => {
+	const { review, review: { product }, siteId, currentStatus, toggleExpanded } = ownProps;
```

Comment:
I carefully looked through mozilla's destructuring assignment page, but i couldn't find an example where the same key was used twice like you did here for review. Are you trying to merge fields into review? I did a little experimenting in the javascript console and was unable to get merging to happen, i.e.:

<img width="308" alt="screen shot 2017-09-22 at 9 43 26 am" src="https://user-images.githubusercontent.com/1595739/30755207-a07195c8-9f7a-11e7-9721-664a1f0e42b6.png">

Reply:
It's to pull another object out of review itself, but I agree it's not clear. I can change this.